### PR TITLE
Added Replicate labels functionality

### PR DESCRIPTION
**Pull request by amitkuagrawal**

### CODE REVIEW

File: src/main/java/com/cognite/sa/beam/replicate/ReplicateLabels.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                .apply("Read source Labels", CogniteIO.readAllLabels()
+                        .withProjectConfig(sourceConfig)
+                        .withHints(Hints.create())
+                        .withReaderConfig(ReaderConfig.create()
+                                .withAppIdentifier(appIdentifier)))
+                .apply("Prepare Labels", ParDo.of(new PrepareLabels()))
+                .apply("Write target Labels", CogniteIO.writeLabels()
+                        .withProjectConfig(targetConfig)
+                        .withHints(Hints.create()
+                                .withWriteShards(20))
```
Comment (kjetilh33) [inline src/main/java/com/cognite/sa/beam/replicate/ReplicateLabels.java:145]: Let's downscale the number of shards. The labels data set is usually quite small, so we can go with 2 shards or so.

File: src/scripts/sh/replicator/RunReplicateLabels-Dataflow.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Delta read override
+export fullRead='true'
+
+#---------------------------------------------------------------------
+# The delta identifier
+#---------------------------------------------------------------------
+export deltaIdentifier='label-replicator'
+
+mvn compile exec:java -Dexec.mainClass=com.cognite.sa.beam.replicate.ReplicateLabels -Dexec.args="--cdfInputSecret=${cdfInputSecret} --cdfInputHost=${cdfInputHost} --cdfTargetSecret=${cdfTargetSecret} --cdfTargetHost=${cdfTargetHost} --project=${gcpProject} --runner=DataFlowRunner --gcpTempLocation=${gcpBucketPrefix}/temp --stagingLocation=${gcpBucketPrefix}/stage/replicate/replicate-labels --region=europe-west1 --experiments=shuffle_mode=service --numWorkers=2 --maxNumWorkers=2 --experiments=enable_stackdriver_agent_metrics --workerMachineType=e2-standard-2"
+
```
Comment (kjetilh33) [inline src/scripts/sh/replicator/RunReplicateLabels-Dataflow.sh:21]: Also here, lets set numWorkers=1 as the labels data set is expected to be small

**Review by kjetilh33 — CHANGES_REQUESTED** [review]

File: src/main/java/com/cognite/sa/beam/replicate/ReplicateLabels.java (hunk 1 above)
Comment (amitkuagrawal) [reply src/main/java/com/cognite/sa/beam/replicate/ReplicateLabels.java]: ok

File: src/scripts/sh/replicator/RunReplicateLabels-Dataflow.sh (hunk 2 above)
Comment (amitkuagrawal) [reply src/scripts/sh/replicator/RunReplicateLabels-Dataflow.sh:21]: ok

**Review by kjetilh33 — APPROVED** [review]

**Pull request merged**